charts/nfs-server-provisioner/values.yaml.gotmpl

```diff
@@ -0,0 +1,14 @@
+persistence:
+  enabled: true
+  storageClass: standard
+  size: 500Gi
+storageClass:
+  name: {{ .Values.namespace }}-nfs
+  provisionerName: cluster.local/{{ .Values.namespace }}-nfs
```

Comment:
is this a convention?

Reply:
For the _cluster.local_ part it seems to be yes, it is also the prefix in the default configuration.

I put the the part after the `/` in order to know how the provisionner will be called.